routes/api/category-routes.js

```diff
@@ -1,28 +1,93 @@
-const router = require('express').Router();
-const { Category, Product } = require('../../models');
+const router = require("express").Router();
+const { Category, Product } = require("../../models");
 
 // The `/api/categories` endpoint
 
-router.get('/', (req, res) => {
-  // find all categories
-  // be sure to include its associated Products
+router.get("/", (req, res) => {
+  Category.findAll({
+    include: {
+      model: Product,
+      attributes: ["id", "product_name", "price", "stock", "category_id"],
+    },
+  })
+    .then((categoryData) => res.json(categoryData))
```

Comment:
consider adding some addition logic to cover more scenarios e.g. if there are no categories present, or if a user is requesting information for category with id 400 but such doesn't exist - this will take your app to the next level :D
Here's an example of some logic you could add to verify if there are any categories present and respond appropriately:
```    
if (!categoryData.length) {
      return res.status(404).json({
        error: "No categories found"
      });
    }

```